### PR TITLE
Remove unused parameter in InstructorCoursesDetailsPage.java #6441

### LINKED ISSUE #6441: Remove unused parameter in InstructorCoursesDetailsPage.java
In `teammates.test.pageobjects.InstructorCoursesDetailsPage#getStudentCountForCourse`, the only parameter `courseId` is not used and thus can be removed.

### PR DESCRIPTION
**Pull request by sponde25**
Fixes #6441

### CODE REVIEW

**Comment by sponde25** [comment]
Need help with resolve conflicts.

File: .settings/com.google.appengine.eclipse.core.prefs [hunk 1]
```diff
@@ -1,8 +1,9 @@
 eclipse.preferences.version=1
-filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.27.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.27.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
+filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.37.jar|appengine-api-labs.jar|appengine-endpoints-deps.jar|appengine-endpoints.jar|appengine-jsr107cache-1.9.37.jar|datanucleus-appengine-1.0.10.final.jar|datanucleus-core-1.1.5.jar|datanucleus-jpa-1.1.5.jar|geronimo-jpa_3.0_spec-1.1.1.jar|geronimo-jta_1.1_spec-1.1.1.jar|jdo2-api-2.3-eb.jar|jsr107cache-1.1.jar
```
Comment (AnaghHegde) [inline .settings/com.google.appengine.eclipse.core.prefs]: You want to say anything about this ?.

**Comment by AnaghHegde** [comment]
@sponde25 
>Need help with resolve conflicts.

You need to keep your fork sync with the master repo. You need to follow every steps [given](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md) here.

File: .settings/com.google.appengine.eclipse.core.prefs (hunk 1 above)
Comment (sponde25) [inline .settings/com.google.appengine.eclipse.core.prefs]: I am not exactly sure why this happened. There was some issue with run configurations not showing local tests. So i ran "gradle setup" again and refreshed the workspace in eclipse. This happened then.  This is related to my eclipse configuration I guess.  I did follow all the steps given in process.md very carefully. Should I give it a go again or you have some other suggestion for me?

**Pull request closed without merging**